view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js

```diff
@@ -100,6 +100,20 @@ define(
             getAdyenBillingAgreements: function () {
                 var self = this;
 
+                // shareable adyen checkout component
+                var checkout = new AdyenCheckout({
```

Comment:
how is this shared ?

Reply:
never mind is clear. Agree better like this.

Reply:
All the hpp payment methods that we render in the `_.map(window.checkoutConfig.payment.adyenOneclick.billingAgreements, function (value) {` van use this component